addons/mail/static/src/core/web/@types/models.d.ts

```diff
@@ -1,15 +1,36 @@
 declare module "models" {
-    export interface Discuss  {
-        inbox: Thread,
-        stared: Thread,
-        history: Thread,
+    export interface Activity {
+        dateCreateFormatted: Readonly<string>;
+        dateDeadlineFormatted: Readonly<string>;
+        dateDoneFormatted: Readonly<string>;
+        edit: () => Promise<object>;
```

Comment:
I don't think we care about the return value, we should probably await the promise rather than return it.